en/02_Developer_Guides/00_Model/04_Data_Types_and_Casting.md

```diff
@@ -50,6 +50,7 @@ Most `DBField` subclasses will be validated using a [`FieldValidator`](api:Silve
 | `Enum` | [`DBEnum`](api:SilverStripe\ORM\FieldType\DBEnum) | An enumeration of a set of strings that can store a single value | Must be one of the defined values |
 | `Float` | [`DBFloat`](api:SilverStripe\ORM\FieldType\DBFloat) | A floating point number | Must be numeric |
 | `ForeignKey` | [`DBForeignKey`](api:SilverStripe\ORM\FieldType\DBForeignKey) | A special `Int` field used for foreign keys in `has_one` relationships | Must be an int |
+| `GeneratedColumn` | [`DBGeneratedColumn`](api:SilverStripe\ORM\FieldType\DBGeneratedColumn) | A column which has its value generated by the database itself, usually based on other column values | N/A |
```

Comment:
I'm not sure if we need this entry if we're using an array syntax, because we could use the keyword 'Generated' / 'GeneratedColumn'

If we do need it, change it to 'Generated' / 'DBGenerated'

Reply:
Updated to `Generated`